### ISSUE TITLE
sacr div cards not showing up, even after reinstall by deleting and redownloading the whole programm incl capture folder

### ISSUE THREAD

**Issue by Gisbert12843** [esge/PoE-HarvestVendor#70]
worked a few weeks ago, doesnt anymore, most other craft do work

**Comment by esge** [comment]
did anything changed in your setup since then? new monitor or so?
could you provide screenshot of the craft for testing?

**Comment by Gisbert12843** [comment]
I did test this quite a bit, did some restarts, redownloads and deleted the temp folder etc

It could be that I changed window settings to Fullscreen and reverted them back to windowed Fullscreen before that

For some reason it fixed itself after a day though 😂

I was actually wondering where it saves my credentials since those are always there no matter where I redownload and unpack the tool..

**Comment by esge** [comment]
yeah, as with pretty much all overlay tools, it doesnt work if you are in fullscreen, has to be windowed or borderless window

settings and stuff is stored in appdata/roaming/poe-harvestvendor
if you go to settings there is a button to open settings folder, it takes you there
